### DISCUSSION TITLE
Suggestion - Add a new common column in QSO Table (log database) indicating with letter the receive status of QSL (Paper/Lotw/Clublog/etc)

### DISCUSSION THREAD

**Discussion by rsaue** [foldynl/QLog#163] [Ideas]
Hi Ladislaw,
Please forgive me if I have overlooked something and the feature I am requesting is already in QLog. 

I suggest you add a new column i log database indicating (like in the DXCC-Tab) with a letter the QSL receive status of all QSL-services the user uses. Looking at the QSO TABLE using smaller monitors this would be useful as it saves screen space. Even with bigger monitors, the user would know the qsl resceive status with a glance at the QSO Table. Users that do not need or want this feature could choose to display the QSL columns as of today. 

Today, the log has one column for each QSL service (QSLr, LoTWr, e.t.c.). CQRLOG has in addition a column indication the feature I request.

Two screenshot showing my log of QSO with VK2GR and EA8DHV in QLog and Cqrlog, shows what I am suggest also for QLog. 

![Screenshot from 2023-01-16 00-27-19](https://user-images.githubusercontent.com/22942440/212573618-59d4eeee-7fd7-44d6-8cdf-6fd95d01d09d.png)

![Screenshot from 2023-01-16 00-29-21](https://user-images.githubusercontent.com/22942440/212573700-cbcf8644-bd5d-4381-93f3-b31eecdb4c79.png)

73 de LB2EG Richard

**Comment by foldynl** [comment]
I try to develop the QLog a little differently. So some things are done differently there. 

In your case, there is a quick overview of the QSL status. see https://github.com/foldynl/QLog/wiki/QSO-Detail#callsign-tooltip

You can also reorder the column in the main windows so you can place QSL columns close to each other

**Reply by rsaue** [reply]
Yes, The tool tip is a nice feature. 

I will keep it in mind using the log. 

Maybe you could add the tool tip function to the Alert spots, or the bit more simplistic version provided in the DX cluster window?

73 de LB2EG Richard

**Reply by foldynl** [reply]
> Maybe you could add the tool tip function to the Alert spots, or the bit more simplistic version provided in the DX cluster window?

DX Cluster contains tooltips but it contains only DX's country. I don't know exactly what the DXC's tooltip should contain. please could you describe your idea in more detail?

Alert spot is a different matter. I tested using tooltips for notifications. But it wasn't very good, because it was chaotic during the high-rate alerts. Therefore, instead of the tooltip, the last alert is displayed on the toolbar (bottom of the main window).

**Reply by rsaue** [reply]
I shall use the Qlog as my main logger when in Linux for a while, and reply back to you if I have a better idea in this area. 

As long as the DX Custer Spot matrix does not include DXCC country, I kind of like the simplistic tooltip there.

Richard
